src/test/java/sonia/scm/jenkins/JenkinsRepositoryHookHandlerTest.java

```diff
@@ -24,71 +24,111 @@
 
```

Comment:
Maybe add a negative Test when rubbish is being send to the server ?

Reply:
How should this test look like? The SCM-Manager cannot check if the parameters are valid.